### PR TITLE
Update import.md

### PR DESCRIPTION
**Pull request by almargit**

### CODE REVIEW

File: learn/classes/import.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
   
 ## Imports and Attributes  
 
-The import statement in Python allows us to use external modules and packages that provide additional functionality. Attributes in Python refer to the properties or values associated with an object that we can access and modify.
+The import [statement in Python](https://diveintopython.org/learn/statements) allows us to use external modules and [packages](https://diveintopython.org/learn/modules/packages) that provide additional functionality. Attributes in Python refer to the properties or values associated with an object that we can access and modify.
```
Comment (angarsky) [inline learn/classes/import.md:8]: Remove new links.

File: learn/classes/import.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-We can use the `import` keyword in Python to import external modules and packages that provide additional functionality. For instance, to use the `math` module in Python, we can write:
+We can use the `import` statement in Python to import external modules and packages that provide additional functionality. For instance, to use the `math` module in Python, we can write:
 
 ```python
 import math
 print(math.pi)
 ```
 
-Once we import the module, we can use its functions and attributes.
+Once we import the module, we can use its [functions](https://diveintopython.org/learn/functions) and attributes.
```
Comment (angarsky) [inline learn/classes/import.md:19]: Remove new links.

File: learn/classes/import.md [hunk 3]
```diff
@@ -37,23 +37,23 @@ Here, we define a class `Circle` that represents a circle with a given `radius`.
...
+In Python, it's possible to import multiple classes from a module by listing the class names and separating them with commas.
 
-Here we import `class1`, `class2`, and `class3` from `module_name`: `from module_name import class1, class2, class3`
+For example, you can import class1, class2, and class3 from the module named `module_name` as follows: `from module_name import class1, class2, class3`.
 
-With the help of `from module_name import *`, we import all the classes from `module_name`. However, this approach is not recommended as it can create naming conflicts if multiple classes with the same name exist in different modules.
+Alternatively, you can use the `from module_name import *` syntax to import all classes from module_name. However, it's worth noting that this approach is discouraged because it can lead to naming conflicts, especially when multiple classes with the same name exist in different modules.
 
-## Importing All Classes from a Module  
+## Importing All Classes from a Module Using From Command
```
Comment (angarsky) [inline learn/classes/import.md:56]: Maybe "using the From Command"? Or "with the From Command".

**Review by angarsky — CHANGES_REQUESTED** [review]
Thank you for your contribution! But some changes are requested.

**Comment by almargit** [comment]
Thank you for your feedback. I have made some adjustments to the pull request in accordance with your notes, as well as a couple of minor changes. Please take a look.

File: learn/classes/static_class.md [hunk 4]
```diff
@@ -118,6 +120,76 @@ print(Math.multiply(2, 3)) ### Output 6
 ```
 
 In the above examples, we can see how to define and call a static method using the `@staticmethod` decorator in Python.  
+
+## When to Use Static Methods
+
+Static methods in Python are a powerful feature that can significantly improve code organization and maintainability. However, it's essential to understand when and where to use them effectively. Here are some common scenarios where static methods can be beneficial:
+
+1. Utility Functions
```
Comment (angarsky) [inline learn/classes/static_class.md:128]: Let's switch to H3 `###` headings + next ones as well.

File: learn/classes/import.md [hunk 5]
```diff
@@ -148,4 +152,40 @@ Suppose we have a package called `example` that contains a submodule `utils` wit
...
-After importing the class, we can create an instance of it by calling the constructor with parentheses.  
+After importing the class, we can create an instance of it by calling the constructor with parentheses.
+
+## Importing Classes from a Parent Directory with `sys.path`
+
+The `sys` module in Python provides access to various system-specific parameters and functions. The `sys.path` list is one of the attributes in this module and contains the directories where Python searches for modules. To import classes or modules from a directory outside the default search path, you can manipulate this list.
+
+Here's a step-by-step guide to importing all classes from another folder in the parent directory:
+
+1. Identify the Target Directory: First, you need to identify the directory from which you want to import classes. This directory should be in the parent directory of your current script.
```
Comment (angarsky) [inline learn/classes/import.md:163]: Let's switch to H3 headings or highlight `Identify the Target Directory` text with bold. + Further items as well.

File: learn/classes/static_class.md [hunk 6]
```diff
@@ -242,6 +314,8 @@ Static variables are shared among all instances of a class and can be accessed u
 
 ## The Difference between a Class Method and a Static Method
 
+Now that you're familiar with class methods and static methods, it's time to delve into the world of static class python and its distinctive properties.
```
Comment (angarsky) [inline learn/classes/static_class.md:317]: 1. Python should start from upper-case.
2. "static class python" doesn't sound good. Let's rephrase it.

**Review by angarsky — CHANGES_REQUESTED** [review]
Added new notes regarding the changes. Please, review them.

**Review by angarsky — APPROVED** [review]
Good job! Let's publish it.

**Pull request merged**